EIPS/eip-1559.md

```diff
@@ -32,34 +34,61 @@ Ethereum currently prices transaction fees using a simple auction mechanism, whe
+
+An important aspect of this upgraded fee system is that miners only get to keep the tips. The base fee is always burned (i.e. it is destroyed by the protocol). Burning this is important because it prevents miners from manipulating the fee in order to extract more fees from users. It also ensures that only ETH can ever be used to pay for transactions on Ethereum, cementing the economic value of ETH within the Ethereum platform. Additionally, this burn counterbalances Ethereum inflation without greatly diminishing miner rewards.
 
-An important aspect of this upgraded fee system is that miners only get to keep the tips. The BASEFEE is always burned (i.e. it is destroyed by the protocol). Burning this is important because it prevents miners from manipulating the fee in order to extract more fees from users. It also ensures that only ETH can ever be used to pay for transactions on Ethereum, cementing the economic value of ETH within the Ethereum platform.
+The transition to this gas price system will occur in two phases, in the first phase both legacy and EIP1559 transactions will be accepted by the protocol. Over the course of this first phase the amount of gas available for processing legacy transactions will decrease while the amount of gas available for processing EIP1559 transactions will increase, moving gas from the legacy pool into the EIP1559 pool until the legacy pool is depleted and the EIP1559 pool contains the entire gas maximum. After all of the gas has transitioned to the EIP1559 pool, the second- finalized- phase is entered and legacy transactions will no longer be accepted on the network.
 
 
 ## Specification
 <!--The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Ethereum platforms (go-ethereum, parity, cpp-ethereum, ethereumj, ethereumjs, and [others](https://github.com/ethereum/wiki/wiki/Clients)).-->
 **Parameters**
-* `FORK_BLKNUM`: TBD
+* `INITIAL_FORK_BLKNUM`: TBD
 * `BASEFEE_MAX_CHANGE_DENOMINATOR`: 8
-* `SLACK_COEFFICIENT`: 3
-* `TARGET_GASUSED`: 8,000,000
+* `TARGET_GAS_USED`: 10,000,000
+* `MAX_GAS_EIP1559`: 16,000,000
+* `FINAL_FORK_BLKNUM`:  `INITIAL_FORK_BLKNUM + (MAX_GAS_EIP1559 / (10 * SLACK_COEFFICIENT))`
```

Comment:
Slack coefficient was removed, but is still used here.

Also, why a slack coefficient of only 1.6?

Reply:
gogo I want to see ethereum at #1

Reply:
Sorry I missed this comment when it was made.

I've removed any mention of the slack coefficient from the doc, as it had already been removed from the implementation.

The slack coefficient was never being used for anything other than as the scalar that described the ratio between max gas used and target gas used. It was 2 when target gas used was 8m and max gas used was 16m, but after switching to a target gas used of 10m it became 1.6.